scss/titles.scss

```diff
@@ -0,0 +1,17 @@
+.title-container { 
```

Comment:
You don't need this, try with `pseudo elements`. Check this [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)